src/main/java/com/google/api/generator/engine/escaper/CommentEscaper.java

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.engine.escaper;
+
+import com.google.common.escape.Escaper;
+import com.google.common.escape.Escapers;
+
+public class CommentEscaper {
+  private static class SpecialEscaper extends Escaper {
+    private static final Escaper escaper =
+        Escapers.builder()
```

Comment:
What is the reference list for these escape characters, can we add a link?

Any reason for handling only a subset of Java's escape characters?

Reply:
The escape [characters](https://stackoverflow.com/questions/1367322/what-are-all-the-escape-characters) in comment would mess the JavaFormmater, so we handle them here. `"`  is overlooked because unlike the string object value, the comment won't be surrounded by `"`, so `\"` in input string is enough.

Reply:
As above, could we please add a link (to Oracle's docs) in the comments of the source code? Also, let's add a comment on why we won't handle certain characters like quotes.

Reply:
of course, added the [link](https://docs.oracle.com/javase/tutorial/java/data/characters.html) for reference, and explained why we do not escape `"` or `'` here. Thanks